composer.json

```diff
@@ -21,17 +21,22 @@
   ],
   "require": {
     "php": ">=5.5.9",
-    "guzzlehttp/guzzle": "^6.2"
+    "guzzlehttp/guzzle": "^6.2",
```

Comment:
Please bump version to 6.2.1, see https://snyk.io/vuln/SNYK-PHP-GUZZLEHTTPGUZZLE-70104